regression/goto-analyzer/variable-sensitivity-bit-field-constants/test.desc

```diff
@@ -0,0 +1,7 @@
+CORE
+main.c
+--function main --variable --structs
+^EXIT=0$
+^SIGNAL=0$
+^bs.*\(\).*->.*\{\.byte=00001010.*@.*,.*\.b0=1.*@.*\}.*$
```

Comment:
Non-blocking, but you may prefer to:

```
__CPROVER_assert(bs.bo==1);
```

Then run with `--verify` and check for:

```
Assertion .* SUCCESS
```

As a slightly less fiddly way to validate this.

Reply:
@thk123 you beat me to it!  Yes, that is a better way of recognising precision at a given point.

Reply:
Ah, cool - I like that, thanks. A bit more knowledge squirrelled away for future reference...